middlewares/auth/forward_test.go

```diff
@@ -229,10 +257,9 @@ func Test_writeHeader(t *testing.T) {
 				req.Host = ""
 			}
 
-			forwardReq := testhelpers.MustNewRequest(http.MethodGet, "http://foo.bar", nil)
+			forwardReq := testhelpers.MustNewRequest(http.MethodGet, "http://foo.bar/path?q=1", nil)
 
 			writeHeader(req, forwardReq, test.trustForwardHeader)
-
```

Comment:
Could you restore this line?

Reply:
Of course. Sorry for this.

Can you tell me which tool will handle such formatting? `go fmt` does not. Or was it just manual checking? Thanks.

Reply:
It's `gofmt` on our CI: https://semaphoreci.com/containous/traefik/branches/pull-request-2398/builds/3

Reply:
Thank you. I see and will check all those validation tests you integrated. Sadly `validate-gofmt` doesn't producde the error when the blank line is missing (Go 1.9.2).

I will check those validation results on GitHub next time.